plugins/modules/aws_secret.py

```diff
@@ -334,7 +334,7 @@ def main():
             'description': dict(default=""),
             'kms_key_id': dict(),
             'secret_type': dict(choices=['binary', 'string'], default="string"),
-            'secret': dict(default=""),
+            'secret': dict(default="", no_log=True),
```

Comment:
This should have a `security_fixes` changelog fragment, and potentially a CVE (or add this to an existing?).